nixos/modules/services/matrix/conduwuit.nix

```diff
@@ -0,0 +1,280 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.conduwuit;
+  defaultUser = "conduwuit";
+  defaultGroup = "conduwuit";
+
+  format = pkgs.formats.toml { };
+  # TOML does not allow null values, so we just omit those fields
+  filteredSettings = lib.converge (lib.filterAttrsRecursive (_: v: v != null)) cfg.settings;
```

Comment:
It'd probably be better to `apply` this to the option itself rather than keep this modification scoped locally

Reply:
Good idea, thanks!